app/src/main/java/com/ncnf/organization/OrganizationBuilder.java

```diff
@@ -0,0 +1,74 @@
+package com.ncnf.organization;
+
+import android.util.Log;
+
+import com.google.firebase.firestore.GeoPoint;
+import com.ncnf.database.DatabaseResponse;
+import com.ncnf.database.DatabaseService;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import static com.ncnf.Utils.ADDRESS_KEY;
+import static com.ncnf.Utils.ADMIN_KEY;
+import static com.ncnf.Utils.DEBUG_TAG;
+import static com.ncnf.Utils.EVENTS_COLLECTION_KEY;
+import static com.ncnf.Utils.LOCATION_KEY;
+import static com.ncnf.Utils.NAME_KEY;
+import static com.ncnf.Utils.ORGANIZATIONS_COLLECTION_KEY;
+import static com.ncnf.Utils.ORGANIZED_EVENTS_KEY;
+import static com.ncnf.Utils.PHONE_NB_KEY;
+import static com.ncnf.Utils.UUID_KEY;
+
+public class OrganizationBuilder {
+
+
+    private DatabaseService db;
+
+    public OrganizationBuilder(){
+        this.db = new DatabaseService();
+    }
+
+    public OrganizationBuilder(DatabaseService db){
+        if(db == null){
+            throw new IllegalArgumentException("Database is null");
+        }
+        this.db = db;
+    }
```

Comment:
Maybe the class would be simpler to use if the method were static. We can save the burden to instantiate an object in order to build another one.
For example: (new Organization(myDb)).loadFromDB(uuid) // Organization.loadFromDB(myDb, data)